app/src/main/java/com/duckduckgo/app/browser/omnibar/OmnibarEntryConverter.kt

```diff
@@ -18,5 +18,5 @@ package com.duckduckgo.app.browser.omnibar
 
 interface OmnibarEntryConverter {
 
-    fun convertQueryToUrl(searchQuery: String): String
+    fun addQueryToCurrentUrl(currentUrl: String, searchQuery: String): String
```

Comment:
In order to keep the currently selected vertical we now need the current URL, hence the method renaming

Reply:
If the user types wikipedia.org as the query, it is not, contrary to the method name, being added to the current url. This is confusing. Also, we now have calls like `queryUrlConverter.addQueryToCurrentUrl(query, query)`.

This is an important piece of logic so it is important that its purpose and meaning are clear. How about we update it as follows:
* use the original method name but include the vertical as an optional extra param when required `fun convertQueryToUrl(searchQuery: String, vertical: String? = null): String`
* deal with adding the vertical param in the method directly. This feel right - we add the query there and the vertical is a part of the query. The work happening in the rewriter is different.

What do you think?